microsetta_public_api/api/microsetta_public_api.yml

```
@@ -205,6 +205,106 @@ paths:
           description: >
             Either `summary_statistics`, `return_raw`, or both are required to be true.
 
+  '/taxonomy/resources/available':
```

is "resources" needed?

---

Probably not, easy to change later, too.
```suggestion
  '/taxonomy/available':
```